.github/workflows/ci.yml

```diff
@@ -214,7 +219,7 @@ jobs:
               echo "${DIRECTORY}" >> ${GITHUB_ENV}
               echo "EOF" >> ${GITHUB_ENV}
               if [[ "${{ env.VERSION }}" == *develop* || "${{ env.VERSION }}" == *master* || "${{ env.VERSION }}" != [0-9]* ]]; then
-                 export VERSION=1.0-$(git rev-parse --abbrev-ref HEAD)-$(git describe --tags --abbrev=1 2>/dev/null)-${GITHUB_RUN_ATTEMPT}
+                 export VERSION=1.0-$(git rev-parse --abbrev-ref HEAD)-$(git describe --tags --abbrev=0 2>/dev/null):${{ env.COMMIT_HASH }}-${GITHUB_RUN_ATTEMPT}
```

Comment:
Why `--abbrev=1` was changed?

Reply:
And are you sure we can use `:` as a separator here? Maybe simple dash `-` is better, isn't it?